### PR TITLE
Add missing env variable

### PR DESCRIPTION
**Pull request by cesarvarela**
This should get Codecov working again

### CODE REVIEW

**Comment by cesarvarela** [comment]
@kepae

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -100,6 +100,7 @@ jobs:
           MONGODB_MIGRATIONS_CONNECTION_STRING: ${{ secrets.MONGODB_MIGRATIONS_CONNECTION_STRING }}
           GATSBY_REALM_APP_GRAPHQL_URL: ${{ secrets.GATSBY_REALM_APP_GRAPHQL_URL }}
           GATSBY_ROLLBAR_TOKEN: ${{ secrets.GATSBY_ROLLBAR_TOKEN }}
+          INSTRUMENT: 1
```
Comment (kepae) [inline .github/workflows/test.yml:103]: Nit: I would prefer `true` to be both consistent with the other file and more readable.

**Review by kepae — CHANGES_REQUESTED** [review]
Feel free to merge!

**Comment by cesarvarela** [comment]
@kepae updated!

**Review by kepae — APPROVED** [review]

**Pull request merged**